source/features/avoid-accidental-submissions.tsx

```diff
@@ -0,0 +1,27 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLFormElement>): void {
+	if (event.key === 'Enter') {
+		event.preventDefault();
+		select('textarea', event.delegateTarget.form)!.focus();
+	}
+}
+
+function init(): void {
+	delegate(document, 'input#issue_title, input#pull_request_title, input#commit-summary-input', 'keydown', onKeyDown);
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isNewIssue,
+		pageDetect.isCompare,
+		pageDetect.isRepo,
+		pageDetect.isEditingFile
+	],
+	awaitDomReady: false,
```

Comment:
```suggestion
	include: [
		pageDetect.isNewIssue,
		pageDetect.isPR,
		pageDetect.isEditingFile
	],
```

Does this work?

Reply:
No, because `isRepo` is needed for commits when *creating* new files e.g. 
https://github.com/sindresorhus/refined-github/new/main

Checked via https://fregante.github.io/github-url-detection/ : 
![2021-04-19_184538](https://user-images.githubusercontent.com/723651/115265035-82899b80-a13f-11eb-9832-6334f463518b.jpg)

Reply:
Please open an issue there.

Adding `isRepo` will add it to all repo pages (PR's Issues etc)

Reply:
I opened it here: https://github.com/fregante/github-url-detection/issues/68